conf/runner.yaml

```diff
@@ -135,6 +135,69 @@ runs:
               recipients:
                   - cloud-ops@mozilla.com
 
+    # Cloud Services TLS monitoring, all sites that must match intermediate level
+    - targets:
+        #AMO
+        - addon.mozilla.org
+        - addons.mozilla.org
+        - blocklist.addons.mozilla.org
+        - builder.addons.mozilla.org
+        - services.addons.mozilla.org
+        - static.addons.mozilla.net
+        - versioncheck-bg.addons.mozilla.org
+        - versioncheck.addons.mozilla.org
+        - compatibility-lookup.services.mozilla.com
+
+        # Balrog
+        - aus5.mozilla.org
+
+        # Firefox Settings (Kinto)
+        - blocklists.settings.services.mozilla.com
+        - firefox.settings.services.mozilla.com
+        - webextensions.settings.services.mozilla.com
+        
+        # Test Pilot
+        - testpilot.firefox.com
+        - embedly-proxy.services.mozilla.com
+        - universalsearch.testpilot.firefox.com
+        
+        # Miracle (Context Graph collection endpoint)
+        - miracle.services.mozilla.com
+        
+        # Pageshot
+        - pageshot.net
```

Comment:
I don't recognize miracle.services.mozilla.com, but the rest use the aws elb policy, so r+ if that meets the intermediate tls rules.